config/default.json

```diff
@@ -30,7 +31,7 @@
   "pendingTxsTimeoutMinutes": 60,
   "rollbackBlocksCount": 25,
   "maxBlockBatchSize": 900,
-  "defaultNetwork": "itn-rewards-v1",
```

Comment:
Is this really okay? I think maybe you meant to merge this change into a different branch than `master-shelley`?